packages/devextreme/js/ui/text_box/ui.text_editor.base.js

```diff
@@ -493,8 +493,8 @@ const TextEditorBase = Editor.inherit({
     },
 
     _setFieldAria(force) {
-        const { 'aria-label': ariaLabel } = this.option('inputAttr');
-
+        const inputAttributeOption = this.option('inputAttr');
```

Comment:
```suggestion
        const inputAttr = this.option('inputAttr');
```

Reply:
[0c3f424](https://github.com/DevExpress/DevExtreme/pull/27328/commits/0c3f424ee03517e7723add3750e3182b0ebe7eff)

Reply:
in this case it whould be better to use `const { inputAttr } = this.option();` but let leave it as it is now